pkg/configs/tiflash/config.go

```diff
@@ -264,3 +264,15 @@ func getProxyDataDir(dataDir string) string {
 	dataDir = GetDataDir(dataDir)
 	return fmt.Sprintf("%s/proxy", dataDir)
 }
+
+func areSlicesEqual(a, b []string) bool {
+	if len(a) != len(b) {
+		return false
+	}
+	for i, sa := range a {
+		if sa != b[i] {
+			return false
+		}
+	}
+	return true
+}
```

Comment:
add ut for coverage?

Reply:
they should already be covered by other cases?